imports/plugins/core/core/server/methods/shop/updateLanguageConfiguration.js

```diff
@@ -22,12 +22,16 @@ export default function updateLanguageConfiguration(language, enabled) {
   }
   this.unblock();
 
-  const shop = Shops.findOne({
-    _id: Reaction.getShopId()
-  });
+  const shopId = Reaction.getShopId();
+
+  const shop = Shops.findOne({ _id: shopId });
 
   const defaultLanguage = shop.language;
 
+  if (language === defaultLanguage && !enabled) {
```

Comment:
If I use the "all languages" toggle, the default language (English in my case) stays enabled, but I am able to turn off `English` if I toggle it individually.

When I refresh the page, `English` is still enabled, so this seems to just be a UI issue that no error is shown.

I don't think a blocker for this, but if it's an easy fix I'd tackle it now.
![languages mov](https://user-images.githubusercontent.com/4482263/64720182-b211f580-d47e-11e9-98c0-edd60872604f.gif)

Reply:
I'll see if it's easy to show the error.

Reply:
Fixed in https://github.com/reactioncommerce/reaction/pull/5514/commits/f9c63580b95ce21af4a75d5f41632ca14bc9ca73 and fixed for the currency list, too.